v1/firebase/adapter.js

```diff
@@ -40,7 +40,18 @@ function apiGetTrack(trackId) {
 
 function apiGetChannel(channelId) {
 	return apiGet(`channels/${channelId}`).then(snapshot => {
-		return serializeChannel(snapshot.val(), channelId);
+		var channel = serializeChannel(snapshot.val(), channelId);
+
+		// Replace `images` with a single `thumbnail` URL.
+		var images = Object.keys(channel.images);
+		var lastImage = images[images.length - 1];
```

Comment:
Maybe it would be more cost-effective to fix that then, yes.

Also, which properties does an image have? A model is only needed if it has more than a property (in that case the URL), I would say.